internal/manager/cache.go

```diff
@@ -54,7 +56,7 @@ type trackerEntry struct {
 
 const idleTimeThreshold time.Duration = 10 * time.Second
 
-func accessFilter(it *cache.Item) bool {
+func accessFilter(key string, it *cache.Item) bool {
```

Comment:
How is `key` used in this function?

Reply:
Due to the latest change in [go-cache](https://github.com/paddlesteamer/go-cache), filter functions must be of type `filter`: https://github.com/paddlesteamer/go-cache/blob/8516d02047f9c4229e500d1ded7c158b43b78d29/cache.go#L1055

That's why there is an unused parameter. It's the design choice of the `paddlesteamer/go-cache`'s developer (which is me :D) and there's nothing I can do about it haha. 

If you're curious about why the `key` parameter may be necessary, the key is used in inline filter function here: https://github.com/paddlesteamer/cloudstash/blob/c3388f135a5a04f66c32e52fc93bd028f4614a01/internal/manager/background.go#L84-L95